File: src/00/sigma_core/src/ontology_gen/01/clamp_world_y.rs
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/spatial/clamp_world_y.md
#![allow(unused_imports)]
use super::super::L00::*;

pub fn clamp_world_y(y: i32) -> i32 {
    math_clamp(y, 0, WORLD_MAX_Y)
}

File: src/_as/01/clamp_world_y.ts
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/spatial/clamp_world_y.md
import { WORLD_MAX_Y, math_clamp } from "../00/mod";

@inline
export function clamp_world_y(y: i32): i32 {
return math_clamp(y, 0, WORLD_MAX_Y);
}
